src/components/NotFound/NotFoundStyles.js

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
+import { colors } from 'colors';
+
+export const StyledNotFound = styled.div`
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: red;
```

Comment:
Why `red`? I don't think we have `red` color anywhere in our design.
Let's use one of the `<Typography>` variants for the text style. You can see which variants we have defined in [theme.js](https://github.com/DarthVanger/NeuralNotes/blob/master/src/theme.js#L15).

For example in Google Drive they don't use `red`. They use the same text style as for the other texts on their website:
![image](https://user-images.githubusercontent.com/6686679/115620052-5ae03200-a2fd-11eb-9756-f1610a4530f7.png)